### DISCUSSION TITLE
Share multiple albums/folders etc. with a same secret

### DISCUSSION THREAD

**Discussion by racecrew** [photoprism/photoprism#3689] [Ideas]
I would like select multiple albums and share all of them with same secret

![image](https://github.com/photoprism/photoprism/assets/17234283/cadc0cf7-60d6-40c9-a8c4-816e16234812)

**Comment by graciousgrey** [comment]
Many improvements are planned for sharing:

https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+Sharing+in%3Atitle
- https://github.com/photoprism/photoprism/issues/1162

**Reply by lastzero** [reply]
Note you can reuse the same secret. Just not multi select and share in a single operation.
